src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -8,7 +8,7 @@
 // associated with an assembly.
```

Comment:
I think we don't want to change this file.

Reply:
Fixed, missed this.